Upstream: No
Reason: In Qt5.15 QPainterPath isn't included implicitly anymore,
so falkon fails to build w/o this include
diff --git a/src/lib/tools/qztools.cpp b/src/lib/tools/qztools.cpp
index 499b225..04f19b7 100644
--- a/src/lib/tools/qztools.cpp
+++ b/src/lib/tools/qztools.cpp
@@ -25,6 +25,7 @@
 #include <QByteArray>
 #include <QPixmap>
 #include <QPainter>
+#include <QPainterPath>
 #include <QBuffer>
 #include <QFile>
 #include <QDir>
